ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -207,6 +246,8 @@ RnsShell::LayerInvalidateMask  RSkComponentTextInput::updateComponentProps(const
   int mask = RnsShell::LayerInvalidateNone;
   std::string textString{};
   RNS_LOG_DEBUG("event count "<<textInputProps.mostRecentEventCount);
+  caretHidden_ = textInputProps.caretHidden;
+  selectionColor_ = textInputProps.selectionColor;
```

Comment:
we may have to set the cursorPaint object color with selectionColor here

Reply:
Don't we have to check if old value and new values are same ? What if selectionColor is not passed (what is the default value ?)